src/components/navbar/index.tsx

```diff
@@ -94,6 +97,17 @@ const NavBar: React.FC<NavBarProps> = ({ tokens }) => {
       >
         Settings
       </Menu.Item>
+      <Menu.Item
+        onClick={() => {
+          if (asyncRequestIsComplete(tokens)) {
+            dispatch(logout());
+            history.push(Routes.HOME);
+            history.go(0); // force refresh if user is already on home page
```

Comment:
⛏️ We dont _need_ to force refresh anymore, I dont really mind if we do though 🤷